datahub-web-react/src/app/ingest/source/IngestionSourceExecutionList.tsx

```diff
@@ -160,6 +161,17 @@ export const IngestionSourceExecutionList = ({ urn, lastRefresh, onRefresh }: Pr
             key: 'x',
             render: (_, record: any) => (
                 <div style={{ display: 'flex', justifyContent: 'right' }}>
+                    {record.urn && (
+                        <Tooltip title="Copy URN. An URN uniquely identifies an entity on DataHub.">
```

Comment:
Let's say Copy Execution Request Urn specifically. No need for the information after that if this is for debugging.